### DISCUSSION TITLE
Help - How to install this ?

### DISCUSSION THREAD

**Discussion by ToxicUrine420** [aleksey-hoffman/sigma-file-manager#53] [Q&A]
Greetings ; I can not figure out how to install this thing , I am not very advanced in computers , please help me .

**Comment by ToxicUrine420** [comment]
Sorry , please don't bother with responding . I had not scrolled to the very bottom , and hence I did not find the file initially .

**Reply by aleksey-hoffman** [reply]
Glad you figured it out. The new update is coming out in the next hour with some error fixes and new features.
